src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -39,6 +39,11 @@
     "The output includes credentials that you must protect. Be sure that you do not include these credentials in "
     "your code or check the credentials into your source control. For more information, see https://aka.ms/azadsp-cli")
 
+ROLE_ASSIGNMENT_CREATE_WARNING = (
+    "In a future release, this command will NOT create any role assignment by default. (--skip-assignment will be the "
```

Comment:
@jiasli , "not" + "any" is a "double negative".  Change to 

"In a future release, this command will NOT create a role assignment by default..."

Reply:
```suggestion
    "In a future release, this command will NOT create a role assignment by default. (--skip-assignment will be the "
```

Reply:
Thanks @dbradish-microsoft